pom.xml

```diff
@@ -151,24 +151,24 @@ THE SOFTWARE.
       <dependency>
         <groupId>junit</groupId>
         <artifactId>junit</artifactId>
-        <version>4.11</version>
+        <version>4.12</version>
```

Comment:
I feel that you just updated versions without trying to use it locally https://code.google.com/p/powermock/issues/detail?id=531 ?